app/src/main/res/layout/exo_player_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#00000000">
+    <com.google.android.exoplayer2.ui.AspectRatioFrameLayout android:id="@+id/exo_content_frame"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="center">
+        <ProgressBar
+            android:id="@+id/exo_buffering"
+            android:layout_width="50dp"
```

Comment:
It is better to use 4 dp grid https://material.io/design/layout/spacing-methods.html#baseline-grid so maybe 48dp or 52dp?

Reply:
Sure, made the change